libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -13,44 +13,56 @@ VotePacketHandler::VotePacketHandler(std::shared_ptr<PeersState> peers_state,
       vote_mgr_(std::move(vote_mgr)),
       seen_votes_(1000000, 1000) {}
 
-void VotePacketHandler::validatePacketRlpFormat(const PacketData &packet_data) const {
-  if (constexpr size_t required_size = 1; packet_data.rlp_.itemCount() != required_size) {
-    throw InvalidRlpItemsCountException(packet_data.type_str_, packet_data.rlp_.itemCount(), required_size);
-  }
+void VotePacketHandler::validatePacketRlpFormat([[maybe_unused]] const PacketData &packet_data) const {
+  // Number of votes is not fixed, nothing to be checked here
```

Comment:
not true, there must be at least 1 vote present so you can check that

Reply:
fixed